scripting/srccoop.sp

```diff
@@ -80,18 +91,7 @@ void LoadGameData()
 		PrepSDKCall_AddParameter(SDKType_Bool, SDKPass_Plain); // bool bUseLRURetirement
 		if (!(g_pCreateServerRagdoll = EndPrepSDKCall()))
```

Comment:
I see that gamedata has been added for hl2dm for CreateServerRagdoll, but here it's still limited to BM. Intended?

Reply:
I had changed that on my local copy, but didn't push it. It is there now.